packages/desktop-client/src/components/manager/ConfigServer.tsx

```diff
@@ -169,40 +167,44 @@ export function ConfigServer() {
       >
         {currentUrl ? (
           <Button
-            type="bare"
+            variant="bare"
+            aria-label="Stop using a server"
             style={{ color: theme.pageTextLight }}
-            onClick={onSkip}
+            onPress={onSkip}
           >
             Stop using a server
           </Button>
         ) : (
           <>
             {!isElectron() && (
               <Button
-                type="bare"
+                variant="bare"
+                aria-label="Use current domain"
```

Comment:
Ok, you get the idea.. There's a bunch more places where the aria-label is unnecessary. I won't comment on all of them.
```suggestion
```